mobile/lib/components/utility/image_with_overlay.dart

```diff
@@ -8,9 +8,9 @@ class ImageWithOverlay extends StatelessWidget {
     super.key,
     required this.thumbnailPath,
     this.child,
-    this.imageBuilder = Image.network,
     this.titleOverlay,
     this.titleOverlayPadding = const EdgeInsets.all(4),
+    this.imageBuilder = Image.network,
   });
 
   /// Child widget to display centered above the image, BUT ABOVE TITLE OVERLAY.
```

Comment:
Since we're doing a lot of clean-up already, let's make this comment more comprehensive (especially since it's mine I think and I don't like how it sounds lol).
```suggestion
  /// Child widget to display centered on top of the image, BUT ABOVE TITLE OVERLAY.
```